src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/RevertPayments/IRevertPaymentsService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Threading.Tasks;
+using SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.RevertPayments.Types;
+
+namespace SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.RevertPayments
+{
+    public interface IRevertPaymentsService
```

Comment:
We shouldn't have a service per action (verb) otherwise well end up with loads of them.  It should be a service for an entity/object.  So this is a service that acts on Payments so it should be the IPayments service with a Revert method.  Maybe fix the PausePayments service as part of this change so that it is a Pause method on an  IPayments service.